source/d2sqlite3/statement.d

```diff
@@ -84,23 +84,51 @@ private:
     }
 
 package(d2sqlite3):
-    this(Database db, string sql)
+
+	this(Database db, const string sql)
+	{
+		string temp = sql;
+		this(db, temp);
+		// this constructor won't be able to give any indication that
+		// the sql wasn't fully parsed, so make sure it is.
+		assert(temp.length == 0);
+		// the second constructor moves the sql parameter to the
+		// unparsed tail, so use that if you might be preparing 2 statements
+	}
+	
+    this(Database db, ref string sql)
```

Comment:
I don't really understand the point of this change. Why complicate the API this much ?